merlin/study/celeryadapter.py

```diff
@@ -488,7 +604,6 @@ def stop_celery_workers(queues=None, spec_worker_names=None, worker_regex=None):
 
     """
     from merlin.celery import app  # pylint: disable=C0415
-    from merlin.config.configfile import CONFIG  # pylint: disable=C0415
```

Comment:
I think moving this early is what broke the tests

Reply:
I think you're right. I just encountered the same issue with the pylint-errors branch